### PR TITLE
Create Default Font Settings

### PR DESCRIPTION
**Pull request by Delphinus8821**
### Description of the Change
Created a new tab in the CONSTELLATION settings for users to set their preference for the font family and size they would like the views of Constellation to be displayed as. The majority of the views have been updated to use this preference for all of the text in the views.

**Views included:**
Analytic View
Attribute Editor
Conversation View 
Data Access View
Delimited File Importer
JDBC Importer
Layers View
Notes View
Plugin Reporter 
Quality Control View
Scatter Plot
Table View
Welcome View
What's New View 

### Alternate Designs
None realised

### Why Should This Be In Core?
This preference setting allows for users to view Constellation in the font and font size they prefer.

### Benefits
Users can set the default font and size to what they find easier to read.

### Possible Drawbacks
N/A

### Verification Process
Tested opening all views with a variety of different fonts and sizes. 
Also tested opening Constellation without closing the views prior to ensure they updated correctly at program launch.

### Applicable Issues
#1016

### CODE REVIEW

File: CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/font-preferences.html
Comment (antares1470) [inline CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/font-preferences.html:11]: The name of the tab from what I could see was "Application Font" rather than "Font" so this should probably be updated here and in other places in the help (such as table of contents, title of page, etc.) just to avoid confusion.

**Review by antares1470 — CHANGES_REQUESTED** [review]
One minor thing but otherwise it looks really good.

**Review by antares1470 — APPROVED** [review]
Once the merge conflict is resolved it should be good to go

**Review by arcturus2 — CHANGES_REQUESTED** [review]
@Delphinus8821 I had a go to see if I could fix some of the bugs like the font not updating when you hit apply and then got stuck into things and was able to find a way to set the font without having to `setStyle` to all buttons and labels. See my suggestions https://github.com/constellation-app/constellation/pull/1061.

The advantage is that it will allow views to be created without having to explicitly set the font and family making it easier to create views and be compliant.

Let me know what you think.

**Review by arcturus2 — APPROVED** [review]
I'd like to get this into the RC so happy to approve and I continue my PR when we can.

**Pull request merged**
